opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ProjectsController.java

```diff
@@ -167,9 +169,12 @@ public void deleteProject(@PathParam("project") final String projectName)
         // Lastly, remove data associated with the project.
         logger.log(Level.INFO, "deleting data for project {0}", projectName);
         for (String dirName: new String[]{IndexDatabase.INDEX_DIR, IndexDatabase.XREF_DIR}) {
-
-            IOUtils.removeRecursive(Paths.get(env.getDataRootPath() +
-                    File.separator + dirName + File.separator + projectName));
+            java.nio.file.Path path = Paths.get(env.getDataRootPath(),dirName, projectName);
```

Comment:
why the full path for `Path` ?
```
env.getDataRootPath(),di
                     ^^^ missing space
```

Reply:
had to use full path, otherwise it will use `javax.ws.rs.Path`.